Fix a bug related to AppStream:

https://github.com/transmission-remote-gtk/transmission-remote-gtk/issues/95

Patch adapted from upstream source repository:

https://github.com/transmission-remote-gtk/transmission-remote-gtk/commit/89259ff90c703c7fe6768b5317803b9aa5b5ab8c

From 89259ff90c703c7fe6768b5317803b9aa5b5ab8c Mon Sep 17 00:00:00 2001
From: Alan <alan-github@eth0.org.uk>
Date: Tue, 17 Nov 2020 01:26:22 +0000
Subject: [PATCH] fix screenshot and add OARS rating so appdata validates

---
 README.md                                           | 5 ++---
 data/io.github.TransmissionRemoteGtk.appdata.xml.in | 5 +++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/README.md b/README.md
index 4306cc7..d38314b 100644
--- a/README.md
+++ b/README.md
@@ -6,13 +6,12 @@ the Transmission BitTorrent client, using its HTTP RPC protocol.
 
 # DEPENDENCIES
 
-The following packages are required dependencies:
+The following packages are required dependencies (debian/ubuntu).
 
 ```bash
-autoconf-archive appstream-glib intltool
+libgtk-3-dev automake autoconf gcc libgeoip-dev gettext autoconf-archive libappstream-dev appstream-util libcurl4-openssl-dev libjson-glib-dev
 ```
 
-
 # BUILDING
 
 Optionally install `libgeoip` and its headers to see the country
diff --git a/data/io.github.TransmissionRemoteGtk.appdata.xml.in b/data/io.github.TransmissionRemoteGtk.appdata.xml.in
index fc35a8f..1476c2a 100644
--- a/data/io.github.TransmissionRemoteGtk.appdata.xml.in
+++ b/data/io.github.TransmissionRemoteGtk.appdata.xml.in
@@ -7,6 +7,7 @@
   <translation type="gettext">transmission-remote-gtk</translation>
   <name>Transmission Remote</name>
   <developer_name>Transmission Remote Gtk Team</developer_name>
+  <content_rating type="oars-1.1" />
   <summary>Remotely manage the Transmission BitTorrent client</summary>
   <url type="homepage">https://github.com/transmission-remote-gtk/transmission-remote-gtk</url>
   <url type="bugtracker">https://github.com/transmission-remote-gtk/transmission-remote-gtk/issues</url>
@@ -23,8 +24,8 @@
     </ul>
   </description>
   <screenshots>
-    <screenshot height="576" width="1024" type="default">
-      <image>http://eth0.org.uk/~alan/transmission-remote-gtk-1.1.1.png</image>
+    <screenshot height="512" width="973" type="default">
+      <image>https://community.linuxmint.com/img/screenshots/transmission-remote-gtk.png</image>
     </screenshot>
   </screenshots>
   <releases>
